Lib/_pydecimal.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2004 Python Software Foundation.
+# Copyright (c) 2017 Python Software Foundation.
```

Comment:
Usually `Copyright (c) <year>` means `<year>-present`, IMHO this should stay  2004.  and 2017 -> Present.

This avoid churn on file and to keep them updated.